mods/ra/rules/aircraft.yaml

```diff
@@ -33,6 +33,8 @@ BADR:
 		Interval: 2
 	-EjectOnDeath:
 	RejectsOrders:
+    GivesExperience:
```

Comment:
I don't think we have a specific lint test for this, but thats a really good idea.